frontend/src/pages/AuthLogin/AuthLogin.tsx

```diff
@@ -115,7 +141,8 @@ export const AuthLogin: React.FC<{ showSignUp?: boolean }> = ({
             /* hideSignUp={
                 !showSignUp && !(process.env.NODE_ENV === 'development')
               }*/
-            hideSignUp={true}
+            // Hide sign up button unless we are in development mode.
+            hideSignUp={!(process.env.NODE_ENV === 'development')}
```

Comment:
So we modified hideSignUp to always be true during the registration and auth redesign work to override the login component.

We will likely modify this again soon, but we're counting on it always being true (hidden) in this AuthLogin component for now.

**This is only true for this AuthLogin.tsx component. The AuthLoginCreate component is correct and should have the the env var check.

Reply:
Good catch. I have changes ready to commit but would like to jump on a call first.

Reply:
@Matthew-Grayson @nickviola Since this a file I've also modified before, if I can jump on that call too that'd be great, thanks!

Reply:
@ameliav Apologies, we resolved it. Please let me know if you have any feedback.

Reply:
@Matthew-Grayson @nickviola  thanks for meeting with me!
Notes from our call: Moving forward I will be removing unused code rather than commenting it out to align with current Crossfeed processes. Also as discussed, we will need to have further discussions on the footer and also the /registration view with the two tabs.
As far as feedback, this PR looks good to me!